app/models/users.py

```diff
@@ -31,3 +49,24 @@ def avatar(self, size):
             digest,
             size,
         )
+
+    def follow(self, user):
+        if not self.is_following(user):
+            self.followed.append(user)
+
+    def unfollow(self, user):
+        if self.is_following(user):
+            self.followed.remove(user)
+
+    def is_following(self, user):
+        return self.followed.filter(followers.c.followed_id == user.id).count() > 0
+
+    async def followed_posts(self):
+        db = await get_db().__anext__()
```

Comment:
thought: https://stackoverflow.com/questions/14337244/how-to-query-inside-a-class-in-sqlalchemy